### DISCUSSION TITLE
--export-process-filter Filter using complete command

### DISCUSSION THREAD

**Discussion by brs-pt** [nicolargo/glances#2774] [Q&A]
Hi, I was looking for a way to export to json stats of a process but using :
glances  --export-process-filter ".*python3.10.*" --export json --export-json-file /tmp/glances.json
export something like:
"11085.cmdline": "python3.10"
"11085.name": "python3.10"
Instead of:
"11085.cmdline": "python3.10 -mMy_module".
Is there a way to export complete cmdline and filter complete cmdline?

**Comment by nicolargo** [comment]
It's clearly a bug...
